docs/usage/configuration/config_documentation.md

```diff
@@ -3670,14 +3671,63 @@ opentracing:
         false
 ```
 ---
-## Workers ##
+## Worker configuration for main process
 Configuration options related to workers.
+Workers are used to scale horizontally and distribute the load to different processes.
+
+The worker configuration is divided into two parts.
+
+1. These switches for the shared configuration to give the main process
+   the necessary information.
+1. And the [switches for the worker configuration file](#worker-configuration-for-workers-configuration)
+   to configure each worker them self are futher below.
+
+The manual how to configure workers is described in detail in the
+[worker documentation](../../workers.md)
+
+---
+### `worker_replication_secret`
+
+A shared secret used by the replication APIs to authenticate HTTP requests
+from workers.
+
+By default this is unused and traffic is not authenticated.
+
+Example configuration:
+```yaml
+worker_replication_secret: "secret_secret"
+```
+---
+### `start_pushers`
+
+Controls sending of push notifications on the main process. Set to `false`
+if using a [pusher worker](../../workers.md#synapseapppusher). Defaults to `true`.
+
+Example configuration:
+```yaml
+start_pushers: false
+```
+---
+### `pusher_instances`
+
+It is possible to run multiple [pusher workers](../../workers.md#synapseapppusher),
+in which case the work is balanced across them. Use this setting to list the pushers by
+[`worker_name`](#worker_name).
+
+If only one pusher worker is configured, this setting is not necessary.
 
+Example configuration:
+```yaml
+pusher_instances:
+  - pusher_worker1
+  - pusher_worker2
+```
 ---
 ### `send_federation`
 
 Controls sending of outbound federation transactions on the main process.
-Set to false if using a federation sender worker. Defaults to true.
+Set to false if using a [federation sender worker](../../workers.md#synapseappfederation_sender).
```

Comment:
```suggestion
Set to `false` if using a [federation sender worker](../../workers.md#synapseappfederation_sender).
```

Reply:
Same as `pushers`. The logic is the same. If it's on it's `federation_sender_instances` map, you don't this at all. [Here](https://github.com/matrix-org/synapse/blob/develop/synapse/config/workers.py#L172)